paper/paper.md

```diff
@@ -0,0 +1,167 @@
+1. **automatic data handling and analysis pipelines:** With a perfect
+   knowledge about which data to expect and where to search for it, many
+   common workflows in data handling and analysis can be simplified or
+   automated through specialized software.
+
+To make use of these benefits, datasets must first be converted to
+BIDS format. Furthermore, already existing data analysis software must be
+extended to be aware of a BIDS format and how to properly use it.
+
+This is where ``MNE-BIDS`` steps in: Harnessing BIDS and the data analysis
+software MNE-Python [@Agramfort2013], it is the goal of ``MNE-BIDS`` to
+automatically organize raw MEG, EEG, and iEEG data into BIDS format and to facilitate
+their analysis.
+
+For example, starting with a single directory full of datafiles with arbitrary
+naming, ``MNE-BIDS`` can be used to extract present metadata, reorganize the
+files into BIDS format, and write additional metadata. Moreover,
+``MNE-BIDS`` supports conversion from raw data formats that are not BIDS
+compatible into permissible formats. As a result, users can easily convert
+their datasets to BIDS in a matter of minutes, rather than after hours of
+manual labour. All conversions performed by MNE-BIDS are validated with the
+bids-validator for a maximum certainty that the results hold up to the BIDS
+format.
+
+Next to this core functionality, ``MNE-BIDS`` is continuously being extended
+to facilitate the analysis of BIDS formatted data. To name some features, it is
+possible to read a raw BIDS dataset and obtain a Python object, ready for
+analyis with MNE-Python, or to save a T1 weighted anatomical MRI image
+alongside the MEG or EEG data and apply an automatic defacing algorithm for
+anonymization purposes.
+
+Users can easily install ``MNE-BIDS`` on all platforms via `pip` and `conda`
+and its functionality is continuously tested on Windows and Linux.
+Next to three core dependencies for scientific computation (`numpy`, `scipy`),
+and handling of MEG, EEG, and iEEG data (`mne`), ``MNE-BIDS`` has minimal
+dependencies, all of which are optional. The API of the package is stable and
+extensively documented and explained in examples (https://mne.tools/mne-bids/).
+
+As of writing, ``MNE-BIDS`` has received code contributions from 15
+contributors and its user base is steadily growing. Code development is
+active and users are typically receiving support within a few hours of opening
+an issue on our dedicated issue tracker.
+
+MNE-BIDS is used in automated analysis pipelines such as the
+MNE-study-template (https://github.com/mne-tools/mne-study-template) and
+several large institutions such as Martinos and "insert Matt's institution" have started to use MNE-BIDS.
+
+The developer team is excited to improve the state of the art in data handling
+and looking forward to welcoming new contributors and users.
+
+# Acknowledgements
+
+MNE-BIDS development is partly supported by XYZ.
```

Comment:
Add French ANR grant and NIH funding in the US

Reply:
Is this a note to yourself? :-) I don't know which specific lines to include.

As for my funding, I am in touch with @franklin-feingold to figure out how I can acknowledge the funding from the US that I receive via a grant of Russ

Reply:
My contributions to MNE-BIDS are funded by the Academy of Finland, grant 310988.